### ISSUE TITLE
[HW2] How to handle invalid route ?

### ISSUE THREAD

**Issue by zeroFruit** [swsnu/swppfall2019#76]
![image](https://user-images.githubusercontent.com/18715362/65892371-1faba480-e3e1-11e9-8a8e-bd1a510fe637.png)

I have a question about this diagram. Is the arrow from **initial -> Login Page** means that in the case of user login, any route except `/login`, `/articles`, `/articles/create`, `/articles/:id`, `/articles/:id/edit` should be redirected to `/login` ?

**Comment by ktaebum** [comment]
Yes, any invalid url (including `/ (just root)`) should be redirected to `/login` (`/articles` if user is already logged in)

**Comment by th-snu** [comment]
does invalid url include something like `/articles/:id` where id is an invalid article id?

**Comment by ktaebum** [comment]
@th-snu no we will do not test with non-existing article id
